autopush/websocket.py

```diff
@@ -986,6 +1000,9 @@ def finish_webpush_notifications(self, result):
             self.ps.updates_sent[str(notif.channel_id)].append(notif)
             msg = notif.websocket_format()
             messages_sent = True
+            update_cnt += 1
```

Comment:
This count should be cleared after all the messages out of storage have been processed to ensure any later requests to check storage get a fresh starting point.